backend-config/namespace_config_test.go

```diff
@@ -162,7 +206,19 @@ func (server *backendConfigServer) ServeHTTP(resp http.ResponseWriter, req *http
 		_, _ = resp.Write([]byte(`{"message":"Unauthorized"}`))
 		return
 	}
-
+	values := req.URL.Query()
+	for k, v := range values {
+		if k == "updatedAfter" {
+			_, err := time.Parse(updateAfterTimeFormat, v[0])
+			if err != nil {
+				resp.WriteHeader(http.StatusBadRequest)
+				_, _ = resp.Write(([]byte(`{"message":"invalid param for updatedAfter"}`)))
+			}
```

Comment:
```suggestion
	_, err := time.Parse( req.URL.Query().Get("updatedAfter"))
				if err != nil {
				resp.WriteHeader(http.StatusBadRequest)
				_, _ = resp.Write(([]byte(`{"message":"invalid param for updatedAfter"}`)))
			}
```

Reply:
This change the semantics a bit. We want to handle 3 scenarios
1. updateAfter is not given
2. updateAfter is given but the value is not parseable as time where we want to throw an exception 
3. updateAfter is given and is parseable as time. 

req.URL.Query().Get(..) does not throw an error. With the change you proposed, we will not be able to distinguish between 1&2

Reply:
removed redundant parenthesis